src/Plugin.php

```diff
@@ -53,9 +53,10 @@ public static function init() {
    * @implements wp_enqueue_scripts
    */
   public static function enqueueAssets() {
-    if (!is_product()) {
+    if (!is_product() || has_term(Product::CATEGORY_EXCLUDED, 'product_cat')) {
```

Comment:
The logic of the condition is not right. The second condition should not be tested if the first is already negative.

Reply:
@Mauricio-Urrego, in my opinion this condition is correct. 🤔 PHP will not evaluate the second condition if don't have to. So if it's not a product, it won't run the `has_term` condition. I assume this was working correctly before, or?